### PR TITLE
Create local metadata no service experiment fixes #2357

### LINKED ISSUE #2357: Local Metadata Long Cache/No Service Experiment
This experiment will investigate the configuration of having a long cache timeout for metadata, and no remote calls.  Then the only source for metadata is the local parser.  So this configuration is:

1) Long metadata cache timeout
2) Local Metadata Parser
3) No Metadata Service

If this experiment is successful (ie no significant regressions in any content or engagement related metrics) then that may suggest we do not need to build any kind of background refresh into the new bootstrapped addon.

### PR DESCRIPTION
**Pull request by jaredkerim**

### CODE REVIEW

File: addon/Feeds/HighlightsFeed.js [hunk 1]
```diff
@@ -139,10 +139,10 @@ module.exports = class HighlightsFeed extends Feed {
           this.refresh("new metadata is available and we're missing data");
         } else if (state.Highlights.rows.length < (HIGHLIGHTS_LENGTH + TOP_SITES_DEFAULT_LENGTH)) {
           // If the user visits a site and we don't have enough weighted highlights yet, refresh the data.
-          this.refresh("there were not enough sites");
+          this.refresh("new metadata is available and there were not enough sites");
         } else if (Date.now() - this.state.lastUpdated >= UPDATE_TIME) {
           // If the user visits a site & the last time we refreshed the data was older than 15 minutes, refresh the data.
-          this.refresh("the sites were too old");
+          this.refresh("new metadata is available and the sites were too old");
```
Comment (rlr) [inline addon/Feeds/HighlightsFeed.js]: random thought I had recently. There should be a "refresh-lite" that doesn't do any SQL queries for cases like these when metadata becomes available.

File: npm-debug.log.3231489098 [hunk 2]
```diff
@@ -0,0 +1,53 @@
+0 info it worked if it ends with ok
```
Comment (rlr) [inline npm-debug.log.3231489098]: oopsies

File: npm-debug.log.3907574705 [hunk 3]
```diff
@@ -0,0 +1,48 @@
+0 info it worked if it ends with ok
```
Comment (rlr) [inline npm-debug.log.3907574705]: daisies

File: npm-debug.log.3231489098 (hunk 2 above)
Comment (jaredkerim) [inline npm-debug.log.3231489098]: OOOOOPS

File: addon/Feeds/HighlightsFeed.js (hunk 1 above)
Comment (jaredkerim) [inline addon/Feeds/HighlightsFeed.js]: Yeah I've thought about that too, but I think it adds more complexity than it solves

File: experiments.json [hunk 4]
```diff
@@ -253,5 +253,20 @@
       "threshold": 0.2,
       "description": "Show pocket trending stories"
     }
+  },
+  "metadataNoService": {
+    "name": "Metadata with No Remote Service",
+    "active": true,
+    "description": "Disable all calls to the metadata service",
+    "control": {
+      "value": false,
```
Comment (jaredkerim) [inline experiments.json]: Oops almost merged this with control set to true.

**Comment by as-pine-proxy** [comment]
[Treeherder: 3838c6758683aafc3e95031a724301ebf4033730](https://treeherder.mozilla.org/#/jobs?repo=pine&revision=3838c6758683aafc3e95031a724301ebf4033730)

**Pull request merged**